config/webpack.config.prod.js

```diff
@@ -218,7 +221,18 @@ module.exports = {
     new webpack.optimize.UglifyJsPlugin({
       compress: {
         screw_ie8: true, // React doesn't support IE8
-        warnings: false
+        warnings: false,
+        pure_getters: true,
+        unsafe: true,
+        unsafe_comps: true,
+        unused: true,
+        dead_code: true,
+        drop_debugger: true,
+        conditionals: true,
+        drop_console: true,
```

Comment:
Does this mean being sure there's no more `console.log` in production? That's pretty cool if so